zerver/lib/actions.py

```diff
@@ -5008,6 +5008,17 @@ def do_create_realm(
             realm.demo_organization_scheduled_deletion_date = (
                 realm.date_created + datetime.timedelta(days=settings.DEMO_ORG_DEADLINE_DAYS)
             )
+
+        # Changing default permission policy for education organisations
+        if org_type == 30 or org_type == 35:
+            realm.create_stream_policy = 2
+            realm.email_address_visibility = 3
+            realm.invite_to_realm_policy = 2
+            realm.edit_topic_policy = 4
+            realm.move_messages_between_streams_policy = 4
+            realm.user_group_edit_policy = 4
+            realm.invite_to_stream_policy = 4
```

Comment:
I would move this to a separate function and call it `set_education_realm_permissions(realm)`

Reply:
Yes, this should produce more cleaner and modular code. I would make these changes to the code right away. Thanks for pointing this out !